### PR TITLE
feat: Waterfox (Flatpak) support

### PR DESCRIPTION
**Pull request by MelihDarcanxyz**
This patch add support for undetected Waterfox (Flatpak).

### CODE REVIEW

**Comment by haggen88** [comment]
You could add another line for the flatpak installed in /var/lib/flatpak: 

Browser(BROWSER_TYPE_WATERFOX_FLATPAK, "Waterfox (Flatpak)", "/var/lib/flatpak/exports/bin/net.waterfox.waterfox", "/var/lib/flatpak/exports/bin/net.waterfox.waterfox"),

File: usr/lib/webapp-manager/common.py [hunk 1]
```diff
@@ -197,6 +198,7 @@ def get_supported_browsers():
                 Browser(BROWSER_TYPE_FIREFOX,  "Waterfox Classic", "waterfox-classic", "/usr/bin/waterfox-classic"),
                 Browser(BROWSER_TYPE_FIREFOX,  "Waterfox 3rd Generation", "waterfox-g3", "/usr/bin/waterfox-g3"),
                 Browser(BROWSER_TYPE_FIREFOX,  "Waterfox 4th Generation", "waterfox-g4", "/usr/bin/waterfox-g4"),
+                Browser(BROWSER_TYPE_WATERFOX_FLATPAK, "Waterfox (Flatpak)", ".local/share/flatpak/exports/bin/net.waterfox.waterfox", ".local/share/flatpak/exports/bin/net.waterfox.waterfox"),
```
Comment (haggen88) [inline usr/lib/webapp-manager/common.py:201]: You could add another line for the flatpak installed in /var/lib/flatpak: 

Browser(BROWSER_TYPE_WATERFOX_FLATPAK, "Waterfox (Flatpak)", "/var/lib/flatpak/exports/bin/net.waterfox.waterfox", "/var/lib/flatpak/exports/bin/net.waterfox.waterfox"),

Comment (MelihDarcanxyz) [reply usr/lib/webapp-manager/common.py:201]: Yeah, didn't know about it. Thanks for your contribution.

Comment (haggen88) [reply usr/lib/webapp-manager/common.py:202]: It is to cover all flatpak installation options, it does not cause any duplicates.

Comment (MelihDarcanxyz) [reply usr/lib/webapp-manager/common.py:202]: Fixed it anyways. Thanks again.

**Pull request merged**